### ISSUE TITLE
Redraw icon

### ISSUE THREAD

**Issue by nobef** [DanielGausi/Nemp#6]
This is just a suggestion/discussion, but I think the program icon is in an outdated style. Take a look and compare.

![изображение](https://user-images.githubusercontent.com/77186394/208053767-6ed528f6-cae8-4c94-9168-1cc91ea13d02.png)

  Here you can offer your options, if you agree and the author does not mind

**Comment by DanielGausi** [comment]
I am open for suggestions. Actually, the current icon was designed by someone els qho thought that the previous icon was outdated. ;-)

My plan for the next version (or at least one of the next versions) is to increase the compatibility with high resolution displays and "scaling". For this, I'll need new graphics anyway (button graphics, icons in the menu, skin graphics, ...). I don't know where to start exactly, but in this context I may update the program icon as well.

**Comment by nobef** [comment]
@DanielGausi 
I don't really know how, but I'll certainly try. I want to say one thing to those who will do it. Now is the time to simplify and enlarge. There is no need, as before, for small details. It is at first glance easier, but the difficulty lies in originality and recognition. Example, Juventus old and new logo :) 
![изображение](https://user-images.githubusercontent.com/77186394/208962408-5254dcef-0a25-4f27-aad7-03a89b37c158.png)

**Comment by nobef** [comment]
@DanielGausi Or is it also just an icon to scale / enlarge? It just matches the style of the player. I have a full hd screen and it's annoying that the icon is small. Otherwise, no complaints.

**Comment by DanielGausi** [comment]
@jezokshe No, this is not just a scaling issue. The icon doesn't use the full area of the square. The reddish waveform uses (almost) the full width, but the actual height of the "N" is only ~50% - the rest is empty space.
I am not a graphic designer - maybe I'll try one of these AI image generators for that. ^^

**Comment by nobef** [comment]
@DanielGausi I also can’t create my own from scratch, only remake someone else’s
 I also wanted to ask if anyone helps you with the translation into Russian?

**Comment by happen-stance** [comment]
![Image](https://github.com/user-attachments/assets/25537297-52be-4e5b-b4a8-20ffae319f24)

Cc. u/PianistAncient2954
From u/Interesting-Union237

My shot at the icon. Tried to copy it's style somewhat and get it more w/ fluent UI.

**Comment by DanielGausi** [comment]
Just a short feedback: I am currently restructuring the code so that scaling with skins can also be implemented. After that I'll have to take care of all the graphics (background, button graphics, menu items, ...), and in that context I'll also take a closer look at the icon. :)
